.github/docker/Dockerfile.centreon-web-bullseye

```diff
@@ -0,0 +1,130 @@
+FROM centreon.jfrog.io/docker/debian:bullseye AS web_fresh
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN <<EOF
+
+apt update
+
+apt install -y lsb-release ca-certificates apt-transport-https software-properties-common wget gnupg2 curl sudo
+
+echo "deb https://packages.sury.org/php/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/sury-php.list
+wget -O- https://packages.sury.org/php/apt.gpg | gpg --dearmor | tee /etc/apt/trusted.gpg.d/php.gpg  > /dev/null 2>&1
+apt update
+
+curl -LsS https://r.mariadb.com/downloads/mariadb_repo_setup | sudo bash -s -- --os-type=debian --os-version=11 --mariadb-server-version="mariadb-10.5"
+
+#echo "deb https://centreon.jfrog.io/artifactory/apt-23.04-stable/ $(lsb_release -sc) main" | tee -a /etc/apt/sources.list.d/centreon.list
+#echo "deb https://centreon.jfrog.io/artifactory/apt-23.04-testing/ $(lsb_release -sc) main" | tee -a /etc/apt/sources.list.d/centreon.list
+echo "deb https://centreon.jfrog.io/artifactory/apt-23.04-unstable/ $(lsb_release -sc) main" | tee -a /etc/apt/sources.list.d/centreon.list
+wget -O- https://centreon.jfrog.io/artifactory/api/security/keypair/Debian/public | gpg --dearmor | tee /etc/apt/trusted.gpg.d/centreon.gpg > /dev/null 2>&1
+apt update
+
+apt install -y mariadb-server mariadb-client
+
+echo "[server]
+skip-grant-tables
+log_output=FILE
+general_log_file=/var/lib/mysql/queries.log
+general_log=0
+slow_query_log_file=/var/lib/mysql/slow_queries.log
+slow_query_log=1
+innodb_file_per_table
+innodb_flush_method=O_DIRECT
+innodb_log_file_size=4M
+innodb_fast_shutdown=0
+" > /etc/mysql/conf.d/container.cnf
+
+apt clean
+
+mkdir /tmp/debs-centreon
+
+EOF
+
+COPY *.deb /tmp/debs-centreon/
+
+RUN <<EOF
+
+apt update
+
+rm -f /tmp/debs-centreon/centreon-23.04*.deb /tmp/debs-centreon/centreon-central-23.04*.deb
+apt install -y /tmp/debs-centreon/centreon-*.deb
+
+rm -f /usr/share/centreon/.env.local.php
+echo "APP_DEBUG=true" >> /usr/share/centreon/.env
+echo "DEBUG_LEVEL=100" >> /usr/share/centreon/.env
```

Comment:
Debug mode is enable by default ?

Reply:
yes, in order to have error messages when a E2E test fails